README.md

```diff
@@ -1 +1,5 @@
-# GitPlayground
\ No newline at end of file
+# GitPlayground
+
+# COmmit 1
```

Comment:
Remove COmmit 1 and commit 2. It is not needed. Update README with  gitignore documentation